changelog.txt

```diff
@@ -1,3 +1,27 @@
+1.11.2 (2021-05-03)
+------------------------------------------------------------------------
+Change: [Win32] Limit hardware accelerated video driver to OpenGL 3.2 or higher (#9077)
+Change: More improvements to the GUI at different scales (#9075, #9102, #9107, #9133, #9174, #9183)
+Fix: Query windows could be partially drawn initially (#9184)
```

Comment:
this sentence is hard to read. Mainly, I think, because "initially" is at the end. Maybe just remove that word completely?